mobile/src/modules/auth/components/AccountListItem.tsx

```diff
@@ -41,24 +44,38 @@ export class AccountListItem extends React.PureComponent<Props> {
     onPress(account);
   }
 
-  render () {
+  render() {
     const { accountRS = '', balanceNQT = '', type = '' } = this.props.account;
```

Comment:
@Titanishu Does react-native support PropTypes? If so, I highly suggest to use it. For non-required props, one can define default values there!

https://hackernoon.com/are-you-using-proptypes-in-react-native-6067e2e5b526

exists!